ui/v2.5/src/components/Images/ImageRecommendationRow.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FunctionComponent } from "react";
+import { useFindImages } from "src/core/StashService";
+import Slider from "react-slick";
+import { ListFilterModel } from "src/models/list-filter/filter";
+import { getSlickSliderSettings } from "src/core/recommendations";
+import { RecommendationRow } from "../FrontPage/RecommendationRow";
+import { FormattedMessage } from "react-intl";
+import { ImageCard } from "./ImageCard";
+
+interface IProps {
+  isTouch: boolean;
+  filter: ListFilterModel;
+  header: String;
+}
+
+export const ImageRecommendationRow: FunctionComponent<IProps> = (
+  props: IProps
+) => {
+  const result = useFindImages(props.filter);
+  const cardCount = result.data?.findImages.count;
+
+  if (!result.loading && !cardCount) {
+    return null;
+  }
+
+  return (
+    <RecommendationRow
+      className="gallery-recommendations"
```

Comment:
We should probably use a new `image-recommendations` class for images that can inherit all the rules `gallery-recommendations` uses. This would enable users to have the ability to define their own rules specifically for images if they wanted.